docker.md

```diff
@@ -0,0 +1,18 @@
+
+## Building The Site Locally By Docker
+
+```bash
+repath=~/git/project/ # The folder path of repository bitcoinops.github.io
+
+cd ${repath}
+
+git clone https://github.com/bitcoinops/bitcoinops.github.io
+
+docker run -d --name bitcoinops -p 4000:4000 -v ${repath}/bitcoinops.github.io:/root/bitcoinops.github.io -w /root/bitcoinops.github.io ruby:2.6.4-stretch /bin/bash -c "bundle install && make preview"
+
+# open url 127.0.0.1:4000 in browser
+
+docker stop bitcoinops
+# edit file
```

Comment:
Not sure if its notable here, but you dont need to stop in order to edit any of the repo files. The make preview will keep running and monitor for file changes and rebuild (in most cases).

Reply:
> Not sure if its notable here, but you dont need to stop in order to edit any of the repo files. The make preview will keep running and monitor for file changes and rebuild (in most cases).

Thank your reminder.

Reply:
We must use absolute repath cause the docker mechanism.

Year, we can check whether is it deployed success by ``docker logs -f bitcoinops``.

I have optimized the file docker.md .